instance/migrations/0114_auto_20190227_1534.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2019-02-27 15:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+
+
+def forwards_func(apps, schema_editor):
+    """
+    Fills configuration_playbook_name according to the openedx_release
+    """
+    # Redefine function here, because if gets deleted in the future this migration
+    # would fail  otherwise
+    def get_base_playbook_name(openedx_release):
+        old_playbook_releases = [
+            'ginkgo',
+            'hawthorn',
+        ]
+        if any([release_name in openedx_release for release_name in old_playbook_releases]):
+            return 'playbooks/edx_sandbox.yml'
+        return 'playbooks/openedx_native.yml'
+
+    # We get the model from the versioned app registry;
+    # if we directly import it, it'll be the wrong version
+    OpenEdXInstance = apps.get_model("instance", "OpenEdXInstance")
+    db_alias = schema_editor.connection.alias
+    # We only need to update the OpenEdXInstance, because these AppServer
```

Comment:
Since the `configuration_playbook_name` is a field in the `OpenEdXInstance` model, I don't think the below explanation is needed. If you think it is still needed, can you check the below suggestions out?
```suggestion
    # We only need to update the OpenEdXInstance, because AppServers
```

Reply:
Makes sense, I'll remove this comment.